lerna.json

```diff
@@ -16,19 +16,10 @@
   "cacheDir": ".changelog",
```

Comment:
(Not related to this PR)

I think that we should exclude this file from prettier, since it clashes with the how lerna generates it (I suppose so because [it has been prettified and reverted two times](https://github.com/jridgewell/babel/commits/80e502905bc80de0074d823d711b1d49bde43b65/lerna.json))

Reply:
Reverted.